src/components/ebay-icon/template.marko

```diff
@@ -4,7 +4,24 @@
 </if>
 <else-if(data.isInline)>
     <svg w-bind class=data.classes style=data.style ${data.htmlAttributes} ${data.accessibilityAttributes} focusable="false">
-        <defs w-id="defs" w-body if(data.renderDefs)/>
+        <if(data.renderDefs && data.themes)>
+            <defs w-id="defs">
+                <%
+                    var theme;
+
+                    if (process.browser) {
+                        theme = data.themes.filter(Boolean)[0];
+                    } else {
+                        // Actual magic, please do not try to replicate.
+                        var lassoContext = out.global["lasso/LassoRenderContext"];
+                        var flags = lassoContext && lassoContext.data.config.flags;
+                        theme = data.themes[!flags || ~flags.indexOf("skin-ds6") ? 1 : 0];
```

Comment:
`~` magic even

Reply:
Originally this was copied in more and I was trying to save bytes. I could make this check better but I don't think it's a big deal. If your against it then I can update the PR.

Reply:
I'm not a fan since its obscure, but its not an issue.  I just don't see it often.